max_sub_array/max_sub_array.py

```diff
@@ -0,0 +1,12 @@
+def max_sub_array_sum(a, size):
+    max_so_far = -9999999 - 1
+    max_ending_here = 0
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp3706_nvb/max_sub_array/max_sub_array.py
+++ b/tmp/tmp3706_nvb/max_sub_array/max_sub_array.py
@@ -8,5 +8,7 @@
         if max_ending_here < 0:
             max_ending_here = 0
     return max_so_far
+
+
 a = [-2, -3, 4, -1, -2, 1, 5, -3]
 print("Maximum contiguous sum is", max_sub_array_sum(a, len(a)))
```